bitbucket-pipelines.yml

```diff
@@ -5,17 +5,46 @@ definitions:
     docker:
       memory: 3072
   steps:
-    - step: &test
-        name: Test
+    - step: &install
+        name: ⚙️ Install
         script:
-          - FORCE_COLOR=true
           - npm ci
+        artifacts:
+          - node_modules/**
+    - step: &lint
+        name: 🧹 Lint
+        script:
           - npm run lint
           - npm run format
           - npm run check-types
+    - step: &test
+        name: 🧪 Test
+        script:
           - npm run test
+    - step: &deploy-temp-stack
+        name: 🏗️ Test Deployment
+        script:
+          - TMP_STAGE=$(cat /dev/urandom | tr -dc 'a-z' | head -c3 | grep -vE 'prd|stg')
```

Comment:
Should we have a comment explaining what this line does (random 3 letter stage name other than 'prd' or 'stg' or are we assuming anyone looking at this should be able to read it?

Reply:
Yeah that's fair, I'll add a comment